core/src/main/kotlin/cache/data/InteractionData.kt

```diff
@@ -19,11 +19,12 @@ data class InteractionData(
     val id: Snowflake,
     val type: InteractionType,
     val data: ApplicationCommandInteractionData,
-    val guildId: Snowflake,
-    val channelId: Snowflake,
-    val member: MemberData,
```

Comment:
use optionals here

Reply:
Did GitHub mess up here again? Because I don't see any optionals or nullables here

Reply:
They did, in your version those became nullables

Reply:
I think @DRSchlaubi said that "I don't see any optionals or nullables here" because (at least here) the code that had those `guildId`/`channelId`/`member` was removed.

Reply:
Yes I'm aware